### DISCUSSION TITLE
Version UMD de l'outil ?

### DISCUSSION THREAD

**Discussion by mknxdev** [AmauriC/tarteaucitron.js#784] [Ideas]
Bonjour,

Je démarre ce thread car j'aimerais premièrement vous remercier pour ce package qui s'avère bien utile pour gérer toutes ces problématiques de RGPD. _What a great tool!_

Mais surtout, j'aimerais aussi apporter une suggestion aux développeurs de cet outil : il me semble que ça peut être intéressant de fournir une version de cet outil qui puisse s'intégrer avec les outils de dev modernes du web front.
Je pense à notamment aux "module bundlers" comme webpack -que j'utilise beaucoup-, qui changent radicalement l'approche du dev front aujourd'hui, mais qui permettent de gérer un tas de traitements et de tâches redondantes de manière aisée, flexible et puissante.
Cela permettrait notamment d'utiliser les nouvelles syntaxes du langages comme les [import](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Statements/import)s JS et d'inclure cet outil dans les processes de build, de compilation/minification, etc.

En fait pour ça, dans l'idéal il faudrait surtout une version de ce package au format [UMD](https://github.com/umdjs/umd), ce qui permettrait de l'intégrer partout, que ce soit via des simples `<script />` dans une page HTML, comme dans les "module bundlers" via les `import` ou les `require`. 

Je vous laisse le lien de webpack si vous voulez en savoir plus : https://webpack.js.org/

Merci d'avance, ça serait vraiment cool d'avoir une version modulable de cet outil. :pray:

**Comment by Colir** [comment]
Bonjour @AmauriC,
comme l'expose @mekkanix, est-il planifié une release en module ce de superbe script ?

merci

**Reply by AmauriC** [reply]
Prévu de mon côté non mais je serai dispo pour aider si quelqu'un se lance ; )
